daq_lib.py

```diff
@@ -626,39 +626,40 @@ def collectData(currentRequest):
   except KeyError as e:
     logger.error('caught key error in logging: %s' % e)
   if (prot == "vector" or prot == "standard" or prot == "stepVector"):
-    seqNum = int(detector_get_seqnum())
-    comm_s = os.environ["LSDCHOME"] + "/runSpotFinder4syncW.py " + data_directory_name + " " + file_prefix + " " + str(currentRequest["uid"]) + " " + str(seqNum) + " " + str(currentIspybDCID)+ "&"
-    logger.info(comm_s)
-    os.system(comm_s)    
-    if img_width > 0: #no dataset processing in stills mode
-      if (reqObj["fastDP"]):
-        if (reqObj["fastEP"]):
-          fastEPFlag = 1
-        else:
-          fastEPFlag = 0
-        if (reqObj["dimple"]):
-          dimpleFlag = 1
-        else:
-          dimpleFlag = 0        
-        nodeName = "fastDPNode" + str((fastDPNodeCounter%fastDPNodeCount)+1)
-        fastDPNodeCounter+=1
-        node = getBlConfig(nodeName)      
-        dimpleNode = getBlConfig("dimpleNode")      
-        if (daq_utils.detector_id == "EIGER-16"):
-          seqNum = int(detector_get_seqnum())
-          comm_s = os.environ["LSDCHOME"] + "/runFastDPH5.py " + data_directory_name + " " + str(seqNum) + " " + str(currentRequest["uid"]) + " " + str(fastEPFlag) + " " + node + " " + str(dimpleFlag) + " " + dimpleNode + " " + str(currentIspybDCID)+ "&"
-        else:
-          comm_s = os.environ["LSDCHOME"] + "/runFastDP.py " + data_directory_name + " " + file_prefix + " " + str(file_number_start) + " " + str(int(round(range_degrees/img_width))) + " " + str(currentRequest["uid"]) + " " + str(fastEPFlag) + " " + node + " " + str(dimpleFlag) + " " + dimpleNode + "&"
-        logger.info(f'Running fastdp command: {comm_s}')
-        if (daq_utils.beamline == "amx"):                                            
-          visitName = daq_utils.getVisitName()
-          if (not os.path.exists(visitName + "/fast_dp_dir")):
-            os.system("killall -KILL loop-fdp-dple-populate")
-            os.system("cd " + visitName + ";${LSDCHOME}/bin/loop-fdp-dple-populate.sh&")
-        os.system(comm_s)
-      if (reqObj["xia2"]):
-        comm_s = f"ssh -q xf17id2-srv1 \"{os.environ['MXPROCESSINGSCRIPTSDIR']}xia2.sh {currentRequest['uid']} \"&"
-        os.system(comm_s)
+    if daq_utils.beamline != "nyx":
```

Comment:
Do we have to put in something equivalent for nyx?

Reply:
not for now - while I don't encourage it, Kevin insists on doing the processing his own way.

we should bring it up with him and see whether he wants to piggyback on the current AMX/FMX code. he has not wanted to in the past